packages/cli/README-source.md

```diff
@@ -301,9 +301,9 @@ zapier convert 1234 --version 1.0.1 my-app
 
 ## Authentication
 
-Most applications require some sort of authentication - and Zapier provides a handful of methods for helping your users authenticate with your application. Zapier will provide some of the core behaviors, but you'll likely need to handle the rest.
+Most applications require some sort of authentication. The Zapier platform provides core behaviors for several authentication methods that might be used with your application, as well as the ability to configure or customize authentication further for certain methods.
```

Comment:
might change `several authentication methods` to `several common authentication methods`- we do the popular ones and if you do them right, then there's usually no further configuration needed.

Reply:
also re:

> customize authentication further for certain methods.

can't you customize all methods?

Reply:
@xavdid Thanks for the great comments here. Now that I think of it, all methods are customizable to an extent - we've even added additional fields for Basic (although that's not mentioned in this text), plus the test request is always custom. No problem to remove that last bit.